obspy/imaging/waveform.py

```diff
@@ -1300,6 +1308,10 @@ def __sect_init_plot(self):
             else:
                 raise NotImplementedError("sect_orientiation '%s' is not "
                                           "valid." % self.sect_orientation)
+            if self.wiglets:
+                fillfun(time, data, self._tr_offsets_norm[_tr], where=data > self._tr_offsets_norm[_tr], facecolor='blue')
+                fillfun(time, data, self._tr_offsets_norm[_tr], where=data < self._tr_offsets_norm[_tr], facecolor='red')
```

Comment:
Hmm..interesting API but it seems like a good way to get the functionality we desire. And maybe we can start to deprecate the gazillion function parameters we have there.

Maybe better to offer a way to reset the style?

```python
default_style = obspy.core.AttribDict({
    "wiglet_color_1": ...
    ...
})

class WaveformPlotting(object):
    style = default_style.copy()

    @classmethod
    def reset_style(cls):
        cls.style = default_style.copy()
```